cypress/.eslintrc.json

```diff
@@ -0,0 +1,15 @@
+{
+  "env": {
+    "cypress/globals": true
+  },
+  "plugins": [
+    "cypress"
+  ],
+  "extends": [
+    "plugin:cypress/recommended"
+  ],
+  "rules" : {
+    "import/no-extraneous-dependencies": "off",
```

Comment:
this lets us put cypress libraries (like cypress-drag-drop) in devDependencies and import them